packages/compiler-cli/src/ngtsc/typecheck/src/type_emitter.ts

```diff
@@ -10,7 +10,7 @@ import {Reference} from '../../imports';
 
 /**
  * A resolved type reference can either be a `Reference`, the original `ts.TypeReferenceNode` itself
- * or null to indicate the no reference could be resolved.
+ * or null to indicate the no reference could be resolved, or that the reference can not be emitted.
```

Comment:
```suggestion
 * or null to indicate that no reference could be resolved, or that the reference can not be emitted.
```

Or even better start a new sentence?

```suggestion
 * or null.
 * A value of null indicates that no reference could be resolved or that the reference can not be emitted.
```